### PR TITLE
Add new font <select> in index, move all font files to fonts subdirec…

### PR DESCRIPTION
**Pull request by after2400**
…tory, update README with source location of additional fonts.

TODO: fix download to work in Firefox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,167 +1,203 @@
...
-              blob = new Blob([svgString], {"type": "image/svg+xml"});
-              a.href = (window.URL || webkitURL).createObjectURL(blob);
-              a.click();
-            });
-          });
-        </script>
-    </body>
+    // Setup user actions
+    $("input,textarea").on('blur', generateSigns);
+    $("input,textarea").on('keypress', (event) => {
```
Comment (smartin015) [inline index.html]: L178-200 indentation/line separation is a bit weird IMO. Could you please update this to match the style in the rest of the doc?

**Review by smartin015 — CHANGES_REQUESTED** [review]
Lookin' good - I have just one request on styling towards the end and I'll approve it.

Thanks!

**Comment by after2400** [comment]
Fixed (I think). Didn't set PhpStorm to ES6 mode so it didn't like that section of JS.

File: index.html [hunk 2]
```diff
@@ -173,31 +173,27 @@ <h3>Settings:</h3>
...
-    a.href = (window.URL || webkitURL).createObjectURL(blob);
-    a.click();
-    })
-    ;
-    })
-    ;
+        // Setup user actions
+        $("input,textarea").on('blur', generateSigns);
+        $("input,textarea").on('keypress', (event) => {
+            if (event.which === 13
```
Comment (smartin015) [inline index.html]: Nit: this should be
```
if (event.which === 13) {
    generateSigns();
}
```

**Review by smartin015 — APPROVED** [review]
Small nit, otherwise LGTM

**Comment by after2400** [comment]
Sorry I should have noticed that, but in my defense it was around 2:30am. If it's not too late, pull the latest commit. I fixed that and added a couple of lines to fix the download functionality in Firefox.

**Comment by smartin015** [comment]
Awesome, thanks! And no worries, been there before :)

**Pull request merged**